src/GeneratedHydrator/CodeGenerator/Visitor/HydratorMethodsVisitor.php

```diff
@@ -34,6 +36,7 @@
  * Replaces methods `__construct`, `hydrate` and `extract` in the classes of the given AST
  *
  * @author Marco Pivetta <ocramius@gmail.com>
+ * @author Pierre Rineau <pierre.rineau@processus.org>
```

Comment:
I actually almost rewrote fully the file, so my name is relevant here. But if the original authors don't want my name, I'll remove it, but I'll answer only to him.